TddProject/src/com/globant/tdd/amountToText/CheckAmountToText.java

```diff
@@ -0,0 +1,21 @@
+package com.globant.tdd.amountToText;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class CheckAmountToText {
+
+	@Test
+	public void checkAmountToText() {
+		Checks check = new Checks(2523.04);
+		assertEquals("Check amount: ", "Two thousand five hundred twenty-three and 04/100 dollars",

```

Comment:
ok, entendido y corregido.

Date: Thu, 22 Jan 2015 05:52:14 -0800
From: notifications@github.com
To: java-bootcamp@noreply.github.com
CC: marcosadriandalmasso@hotmail.com
Subject: Re: [java-bootcamp] Topic 3 / Práctica 1 (#7)

In TddProject/src/com/globant/tdd/amountToText/CheckAmountToText.java:

> @@ -0,0 +1,21 @@
> +package com.globant.tdd.amountToText;
> +
> +import static org.junit.Assert.*;
> +import org.junit.Test;
> +
> +public class CheckAmountToText {
> +
> +	@Test
> +	public void checkAmountToText() {
> +		Checks check = new Checks(2523.04);
> +		assertEquals("Check amount: ", "Two thousand five hundred twenty-three and 04/100 dollars",

Lo mismo acá, el texto de información debería ser descriptivo de lo que se está testeando.


—
Reply to this email directly or view it on GitHub.

  
    
    
  
  
 		 	   		  =